pkg/xds/generator/testdata/admin/05.envoy-config.golden.yaml

```diff
@@ -62,6 +62,11 @@ resources:
               - '*'
               name: kuma:envoy:admin
               routes:
+              - match:
+                  prefix: /ready
+                route:
+                  cluster: kuma:envoy:admin
```

Comment:
Should it be admin or `kuma_readiness`?

Reply:
Case 05 does not specify a value for `readinessPort`, so this is the case of an existing/old DPP is connecting to a new CP. We'll still forward traffic to the admin API in this scenario because there is no readiness reporter component running in the DP.